src/modules/LR11x0/LR11x0.cpp

```diff
@@ -460,10 +481,9 @@ int16_t LR11x0::startReceive(uint32_t timeout, uint32_t irqFlags, uint32_t irqMa
   // set DIO mapping
   uint32_t irq = irqFlags;
   if(timeout != RADIOLIB_LR11X0_RX_TIMEOUT_INF) {
-    irq |= RADIOLIB_LR11X0_IRQ_TIMEOUT;
+    irq |= (1UL << RADIOLIB_IRQ_TIMEOUT);
   }
-
-  state = setDioIrqParams(irq);
+  state = setDioIrqParams(getIrqMapped(irq));
```

Comment:
Just stumbled upon this: I believe due to `getIrqMapped()` the flags in `startReceive()` on line 442 should be `RADIOLIB_IRQ_RX_DEFAULT_MASK`.

Reply:
Ah, you're right - not sure why I didn't catch this in the testing, but it's fixed in e0af297c76ef09df9fef6afcafe67196ab58957f, thanks!

Reply:
@jgromes I'm sorry but I believe he flags shouldn't be set to `RADIOLIB_IRQ_RX_DEFAULT_FLAGS`, because for the LR11x0 it will then also set the mask to this, so it would generate interrupts on all the default flags. It should set the flags to the default masks (yes, confusing).

Reply:
Good point - since LR11x0 doesn't support the IRQ mask ,we need to the masking ourselves, much like in SX127x: `getIrqMapped(irqFlags & irqMask`